cranelift/filetests/src/subtest.rs

```diff
@@ -107,6 +107,12 @@ pub trait SubTest {
 
 /// Run filecheck on `text`, using directives extracted from `context`.
 pub fn run_filecheck(text: &str, context: &Context) -> anyhow::Result<()> {
+    log::debug!(
+        "Filecheck Input:\n\
+         =======================\n\
+         {text}\n\
+         ======================="
+    );
```

Comment:
Is this debug log message generally useful, or a leftover?

Reply:
It is generally useful. So hard to get the input that the patterns are being filechecked against, now we can always see it via `RUST_LOG=debug`.